stimpacks/deploy/docker.go

```diff
@@ -42,6 +43,11 @@ func (d *Deploy) startDeployContainer(instance *Instance) {
 	if err != nil {
 		d.log.Fatal("Unable to determine home directory. {}", err)
 	}
+	cacheDir := fmt.Sprintf("%s/.kube-vault-deploy/bin-cache", home)
```

Comment:
Should use filepath.Join() for each item so it works in windows

Reply:
True dat.